knora/dsplib/utils/stack_handling.py

```diff
@@ -0,0 +1,67 @@
+import re
+import subprocess
+import time
+
+import requests
+
+from knora.dsplib.models.helpers import BaseError
+
+
+def start_stack() -> None:
+    """
+    Start the Docker containers of DSP-API, and load some basic data models and data.
+    """
+    # start up the fuseki database
+    completed_process = subprocess.run("docker compose up db -d", shell=True, cwd="knora/dsplib/docker")
+    if not completed_process or completed_process.returncode != 0:
+        raise BaseError("Cannot start the API: Error while executing 'docker compose up db -d'")
+    time.sleep(5)
+
+    # inside fuseki, create the "knora-test" repository
+    url_prefix = "https://raw.githubusercontent.com/dasch-swiss/dsp-api/main"
+    repo_template = requests.get(f"{url_prefix}/webapi/scripts/fuseki-repository-config.ttl.template").text
+    repo_template = re.sub(r"@REPOSITORY@", "knora-test", repo_template)
+    response = requests.post(
+        url="http://0.0.0.0:3030/$/datasets",
+        files={"file": ("file.ttl", repo_template, "text/turtle; charset=utf8")},
+        auth=("admin", "test")
+    )
+    if not response.ok:
+        raise BaseError("Cannot start the API: Error when creating the 'knora-test' repository")
+
+    # load some basic ontos and data into the repository
```

Comment:
Sometimes you use "onto", sometimes "ontology", sometimes "data model". I would always use the same term for the same thing.